modules/core/src/utils/typed-array-manager.js

```diff
@@ -9,7 +9,11 @@ export class TypedArrayManager {
     Object.assign(this.props, props);
   }
 
-  allocate(typedArray, count, {size = 1, type, padding = 0, copy = false, initialize = false}) {
+  allocate(
+    typedArray,
+    count,
+    {size = 1, type, padding = 0, copy = false, initialize = false, overAllocCountCap}
```

Comment:
`maxCount` to be more concise

Reply:
✔

Reply:
FYI you can click on "resolve conversation" instead of replying to every comment.